### ISSUE TITLE
Reset doesn't allow a new query to be run

### ISSUE THREAD

**Issue by jewzaam** [lightblue-platform/lightblue-applications#110]
I have a long running query and hit "Reset".  It cleared my current query but the old query is still running.

Maybe add a way to cancel query on the client side?  To at least allow execution of something else...
And while at it, maybe a way to report a slow running query and cancel?

**Comment by alechenninger** [comment]
Good point. This is pretty straightforward to implement it looks like. I'll take a look soon.
